docker/pg-no-ww.Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM perl:5.32
+# set metadata
+LABEL maintainer="fabian.gabel@tuhh.de"
+LABEL hub.docker.com="eltenedor/pg-no-ww"
```

Comment:
These labels should be changed to something that belongs to openwebwork or should be deleted.

Reply:
Agreed.  We should have a openwebwork organization in docker as well.  Should I do now and switch things over?